recipes/obspy/meta.yaml

```diff
@@ -0,0 +1,65 @@
+# {% set version = "1.0.2.dev0" %}
+
+package:
+  name: obspy
+  version: {{ version }}
+
+source:
+#  fn: obspy-{{ version }}.zip
+#  url: https://pypi.python.org/packages/source/o/obspy/obspy-{{ version }}.zip
+  fn: maintenance_1.0.x.zip
+  url: https://github.com/obspy/obspy/archive/maintenance_1.0.x.zip
+
```

Comment:
note that you run that command on your computer, on a known-good file - not in your recipe.